components/google_calendar/google_calendar.app.js

```diff
@@ -1,154 +1,307 @@
       optional: true,
-      type: "string"
+      type: "string",
     },
     timeZone: {
       description: "Time zone used in the response. Optional. The default is the time zone of the calendar.",
       optional: true,
-      type: "string"
+      type: "string",
+      label: "TimeZone",
+    },
+    updatedMin: {
+      description: "Lower bound for an event's last modification time (as a RFC3339 timestamp) to filter by. When specified, entries deleted since this time will always be included regardless of showDeleted. Optional. The default is not to filter by last modification time.",
+      optional: true,
+      type: "string",
+    },
+    ruleId: {
+      type: "string",
+      label: "ACL rule identifier",
+      description: "ACL rule identifier.",
+      async options({ calendarId }) {
+        return await this.listACLOptions(calendarId);
+      },
     },
-    updatedMin: { description: "Lower bound for an event's last modification time (as a RFC3339 timestamp) to filter by. When specified, entries deleted since this time will always be included regardless of showDeleted. Optional. The default is not to filter by last modification time.",
+    role: {
+      label: "Role",
+      type: "string",
+      description: "The role assigned to the scope.",
       optional: true,
-      type: "string"
-    }
+      options() {
+        const roles = [
+          "none",
+          "freeBusyReader",
+          "reader",
+          "writer",
+          "owner",
+        ];
+        const options = roles.map((role) => {
+          return {
+            label: role,
+            value: role,
+          };
+        });
+        return options;
+      },
+    },
+    scopeType: {
+      label: "Type of the Scope",
+      type: "string",
+      description: "The extent to which calendar access is granted by this ACL rule.",
+      optional: true,
+      default: "default",
+      options() {
+        const types = [
+          "user",
+          "group",
+          "domain",
+        ];
+        const options = types.map((type) => {
+          return {
+            label: type,
+            value: type,
+          };
+        });
+        return options;
+      },
+    },
+    scopeValue: {
+      label: "Type of the Scope",
+      type: "string",
+      description: "The email address of a user or group, or the name of a domain, depending on the scope type. Omitted for type 'default'",
+      optional: true,
+      default: "",
+    },
   },
   methods: {
     _tokens() {
-      const access_token = get(this, "$auth.oauth_access_token")
-      const refresh_token = get(this, "$auth.oauth_refresh_token")
+      const access_token = get(this, "$auth.oauth_access_token");
+      const refresh_token = get(this, "$auth.oauth_refresh_token");
       return {
         access_token,
         refresh_token,
-      }
+      };
```

Comment:
You can avoid the eslint error: **Identifier 'access_token' is not in [camel case](https://eslint.org/docs/rules/camelcase).**
```suggestion
      return {
        access_token: this.$auth.oauth_access_token,
        refresh_token: this.$auth.oauth_refresh_token,
      };
```